contributrs-quick-start.rst

```diff
@@ -0,0 +1,976 @@
+
+.. |STATIC_CODE_CHECKS.rst| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst" target="_blank">
+   STATIC_CODE_CHECKS.rst</a>
+
+- Following are some of the important links of STATIC_CODE_CHECKS.rst
+
+  - |Pre-commit Hooks|
+
+  .. |Pre-commit Hooks| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst#pre-commit-hooks" target="_blank">
+   Pre-commit Hooks</a>
+
+  - |Pylint Static Code Checks|
+
+  .. |Pylint Static Code Checks| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst#pylint-static-code-checks"
+   target="_blank">Pylint Static Code Checks</a>
+
+
+  - |Running Static Code Checks via Breeze|
+
+  .. |Running Static Code Checks via Breeze| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst#running-static-code-checks-via-breeze"
+   target="_blank">Running Static Code Checks via Breeze</a>
+
+
+
+
+
+Contribution guide
+-----------------------------------
+
+- To know how to contribute to the project visit |CONTRIBUTING.rst|
+
+.. |CONTRIBUTING.rst| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst" target="_blank">CONTRIBUTING.rst</a>
+
+- Following are some of important links of CONTRIBUTING.rst
+
+  - |Types of contributions|
+
+  .. |Types of contributions| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contributions" target="_blank">
+   Types of contributions</a>
+
+
+  - |Roles of contributor|
+
+  .. |Roles of contributor| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#roles" target="_blank">Roles of
+   contributor</a>
+
+
+  - |Workflow for a contribution|
+
+  .. |Workflow for a contribution| raw:: html
+
+   <a href="https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow" target="_blank">
+   Workflow for a contribution</a>
+
+
+
+Raising Pull Request
+-----------------------------------
+
+1. Go to your Github account and open your fork project and click on Branches
+
+   .. raw:: html
+
+    <div align="center" style="padding-bottom:10px">
+      <img src="images/quick_start/pr1.png"
+           alt="Goto fork and select branches">
+    </div>
+
+2. Click on ``New pull request`` button on branch from which you want to raise a pull request.
+
+   .. raw:: html
+
+      <div align="center" style="padding-bottom:10px">
+        <img src="images/quick_start/pr2.png"
+             alt="Accessing local airflow">
+      </div>
+
+3. Add title and description as per Contributing guidelines and click on ``Create pull request``.
+
+   .. raw:: html
+
+      <div align="center" style="padding-bottom:10px">
+        <img src="images/quick_start/pr3.png"
+             alt="Accessing local airflow">
+      </div>
+
+
+Syncing Fork and rebasing Pull request
+--------------------------------------
+
```

Comment:
Maybe one paragraph of explantation why this is needed? something like:

Often it takes several days or weeks to discuss and iterate with the PR until it is ready to merge. In the meantime new commits are merged, and you might run into conflicts, therefore you should periodically synchronize master in your fork with the `apache/airflow` master and rebase your PR on top of it. This chapter describes how to do it.